eleventy.config.js

```diff
@@ -1,17 +1,101 @@
+const { feedPlugin } = require('@11ty/eleventy-plugin-rss');
+const { indexPostsBy, flatPaginate } = require('./plugins/pagination-helpers');
 const pulsarEleventyConfig = require("11ty-config");
+const { getMdLibrary } = pulsarEleventyConfig;
 
 module.exports = (eleventyConfig) => {
-
   pulsarEleventyConfig(eleventyConfig);
 
+  getMdLibrary().use(
+    require("markdown-it-anchor")
+  );
+
+  // Don't generate JS files when we encounter `*.11tydata.js` files; those are
+  // metadata.
+  eleventyConfig.ignores.add("**/*.11tydata.js");
+
+  eleventyConfig.addPlugin(feedPlugin, {
+    type: "atom", // or "rss", "json"
+    outputPath: "/feed.xml",
+    collection: {
+      name: "blog-posts-feed", // iterate over `collections.posts`
+      limit: 10,     // 0 means no limit
+    },
+    metadata: {
+      language: "en",
+      title: "Pulsar Blog",
+      subtitle: "The blog for Pulsar: a community-led, hyper-hackable text editor.",
+      base: "https://blog.pulsar-edit.dev/",
+      author: {
+        name: "Pulsar Team",
+        email: "", // Optional
```

Comment:
```suggestion
        email: "admin@pulsar-edit.dev",
```

If we have it might as well add it